App/ListJS.js

```diff
@@ -0,0 +1,80 @@
+var playListFromJS= JSON.parse(playlistsJSON);
+
+
+document.addEventListener("DOMContentLoaded", function(event) {
+
+    createPlayLists();
+
+  });
+
+//function Create playlists
+function createPlayLists(){
+  console.log(playListFromJS);
+  for (var i = 0; i < playListFromJS.length; i++) {
+    var bigPlaylist = new Playlist(playListFromJS[i]);
+    drawBigPlayList(bigPlaylist);
```

Comment:
Separate drawing for creating playlists: create them, store them in a collection and then draw them